src/helm/benchmark/adaptation/adapter_spec.py

```diff
@@ -8,6 +8,7 @@
 ADAPT_GENERATION: str = "generation"
 ADAPT_LANGUAGE_MODELING: str = "language_modeling"
 ADAPT_MULTIPLE_CHOICE_JOINT: str = "multiple_choice_joint"
+ADAPT_MULTIPLE_CHOICE_JOINT_COT: str = "multiple_choice_joint_cot"
```

Comment:
optional: I would prefer for cot to be spelled out in full rather than abbreviated.